DQMServices/FileIO/python/DQMFileSaverPB_cfi.py

```diff
@@ -1,7 +1,15 @@
 import FWCore.ParameterSet.Config as cms
 
 # DQM file saver module
-dqmSaver = cms.EDAnalyzer("DQMFileSaverPB",
+
+# Since we are disabling the AnelasticDQM process in the DQM online machines
+# we are going to start accumulating PB files there.
+# The simplest solution to avoid accumulating files is to disable the PB
+# file production in all clients, since the new GUI is not deployes anyway, 
+# by replacing the dqmSaver with a stub dqmSaver.
+# dqmSaver = cms.EDAnalyzer("DQMFileSaverPB",
+
+dqmSaver = cms.EDAnalyzer("DQMFileSaverPBStub",
```

Comment:
@Martin-Grunewald is this going to mess up with the parsing of the GUI?

Reply:
shall we make this

https://github.com/cms-sw/cmssw/blob/cea249650467c0724ea7dadb238cef942570c708/DQMServices/FileIO/plugins/DQMFileSaverPB.cc#L228

make actually write out a cfi with `addWithDefaultLabel`?

Reply:
It would need a proper fillDescriptions method with all the parameters.

Reply:
In the end these are two different plugins and thus need two separate cfi files (as each cfi file contains the class name).